decidim-bulletin_board-js/src/voter/voter_wrapper_dummy.js

```diff
@@ -8,11 +13,64 @@ export const WAIT_TIME_MS = 500; // 0.5s
 export class VoterWrapper {
   constructor({ voterId }) {
     this.voterId = voterId;
+    this.jointElectionKey = null;
+    this.contests = {};
+  }
+
+  /**
+   * Process the message and update the wrapper status.
+   *
+   * @returns {Object|undefined}
+   */
+  processMessage(messageId, message) {
+    const messageIdentifier = MessageIdentifier.parse(messageId);
+    switch (messageIdentifier.type) {
+      case CREATE_ELECTION: {
+        this.contests = message.description.contests;
+        break;
+      }
+      case END_KEY_CEREMONY: {
+        const content = JSON.parse(message.content);
+        this.jointElectionKey = content.joint_election_key;
+        break;
+      }
+    }
   }
 
-  encrypt(data) {
-    return new Promise((resolve) =>
-      setTimeout(resolve, WAIT_TIME_MS)
-    ).then(() => JSON.stringify(data));
+  async encrypt(vote) {
+    return new Promise((resolve) => setTimeout(resolve, WAIT_TIME_MS)).then(
+      () => {
+        if (!this.jointElectionKey) {
+          console.warn("Invalid election status.");
+          return;
+        }
+        return JSON.stringify(this.createBallot(vote));
+      }
+    );
+  }
+
+  createBallot(vote) {
```

Comment:
I know it is just a dummy wrapper but can you add the documentation blocks that are missing for these methods. Also mark them as private if they are 😄